CHANGELOG.md

```diff
@@ -8,11 +8,96 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Add new way to customize the Snap account creation flow ([#31285](https://github.com/MetaMask/metamask-extension/pull/31285))
+- Add icon image for Plume network ([#31712](https://github.com/MetaMask/metamask-extension/pull/31712))
+- [Beta] Solana: update add account from opt in solana ([#31387](https://github.com/MetaMask/metamask-extension/pull/31387))
+- [Beta] Solana: update add account from network picker ([#31358](https://github.com/MetaMask/metamask-extension/pull/31358))
+- Add clear functionality to SRP import error banner ([#30673](https://github.com/MetaMask/metamask-extension/pull/30673))
+- Multichain tokens import ([#31201](https://github.com/MetaMask/metamask-extension/pull/31201))
+- Add remote-mode feature slide ([#31463](https://github.com/MetaMask/metamask-extension/pull/31463))
+- Implement the publishBatch hook for smart transactions ([#31267](https://github.com/MetaMask/metamask-extension/pull/31267))
+- [Beta] Create Solana account automatically on wallet creation or SRP import ([#32038](https://github.com/MetaMask/metamask-extension/pull/32038))
+
+### Changed
+- Bump transaction controller version ([#31440](https://github.com/MetaMask/metamask-extension/pull/31440))
+- Update default account name visibility/width ([#31202](https://github.com/MetaMask/metamask-extension/pull/31202))
+- Updating Text component "body" font sizes ([#31494](https://github.com/MetaMask/metamask-extension/pull/31494))
+- Update popup width of extension from 357px to 400px ([#31443](https://github.com/MetaMask/metamask-extension/pull/31443))
+- Update font family from euclid to centra ([#31303](https://github.com/MetaMask/metamask-extension/pull/31303))
+- Refactor multichain activity tab ([#31439](https://github.com/MetaMask/metamask-extension/pull/31439))
+- Update alignment and hover effect for token cell ([#31615](https://github.com/MetaMask/metamask-extension/pull/31615))
+- Update font weights for balance price and other headers ([#31624](https://github.com/MetaMask/metamask-extension/pull/31624))
+- Update Soneium logo ([#31573](https://github.com/MetaMask/metamask-extension/pull/31573))
+- Add XRPL EVM Testnet network ([#31533](https://github.com/MetaMask/metamask-extension/pull/31533))
+- Update color of values to text-default ([#30886](https://github.com/MetaMask/metamask-extension/pull/30886))
+- Update tabs color hover and animations ([#30907](https://github.com/MetaMask/metamask-extension/pull/30907))
 
 ### Fixed
 - Fix NFT removal on different networks ([#32102](https://github.com/MetaMask/metamask-extension/pull/32102))
-- Fix exchange rate lookups on Solana Swap page to prevent crashing when values are undefined ([#32114](https://github.com/MetaMask/metamask-extension/pull/32114))
+- [Beta] Fix exchange rate lookups on Solana Swap page to prevent crashing when values are undefined ([#32114](https://github.com/MetaMask/metamask-extension/pull/32114))
```

Comment:
Not 100% about this one. should be `fix` maybe as well?
```suggestion
- [Beta] fix exchange rate lookups on Solana Swap page to prevent crashing when values are undefined ([#32114](https://github.com/MetaMask/metamask-extension/pull/32114))
```

Reply:
Good callout, indeed there's no standard style but capitalized